### DISCUSSION TITLE
Difficulty in understanding the assigned value of Pole vector

### DISCUSSION THREAD

**Discussion by KodalAnwesha** [gempy-project/gempy#915] [Q&A]
Hello,
I have some problem regarding the assigned value of pole vector in the tutorial. It gives the dip, azimuth and polarity. Is it assigning the the dip, azimuth and polarity values for the given surface itself? If it is so, then how the polarity value is assigned? What should be the range of values of polarity?
Thanks

**Comment by spaceswimmer** [comment]
I use polarity to flip vectors, so either 1 or -1

**Comment by aradfarahani** [comment]
Hi There
In the context of GemPy, the pole vector is used to describe the orientation of geological surfaces. The pole vector is defined by three parameters: dip, azimuth, and polarity.

### Explanation of Parameters:
1. **Dip**: This is the angle between the horizontal plane and the geological surface, measured downward. It ranges from 0° to 90°.
2. **Azimuth**: This is the compass direction in which the dip is measured. It ranges from 0° to 360°, where 0° represents north, 90° east, 180° south, and 270° west.
3. **Polarity**: This parameter indicates the direction of the pole vector. In GemPy, polarity is typically used to distinguish between different orientations of the same surface. The range of polarity values can vary, but it is often binary (e.g., 1 or -1) to indicate opposite directions³.

### Assigning Values:
- **Dip and Azimuth**: These values are assigned based on the orientation of the geological surface itself. The dip is the angle of inclination, and the azimuth is the direction of this inclination.
- **Polarity**: This value is assigned to indicate the direction of the pole vector. It helps in distinguishing between different orientations of the same surface. For example, a polarity of  1 might indicate one direction, while -1 indicates the opposite direction.

### Example:
If you have a geological surface with a dip of 30°, an azimuth of 120°, and a polarity of 1, it means:
- The surface dips 30° downward.
- The dip direction is 120° (southeast).
- The polarity indicates the specific orientation of the pole vector.
